core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2677,6 +2677,16 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey SECURITY_LOGIN_IMPERSONATION_USERNAME =
+      new Builder(Name.SECURITY_LOGIN_IMPERSONATION_USERNAME).setDescription(String.format(
+          "When %s is set to SIMPLE or CUSTOM, user application uses this property to indicate "
+              + "the IMPERSONATED user requesting Alluxio service. If it is not set explicitly, "
+              + "impersonation will not be used. A special value of '%s' can be specified to "
+              + "impersonate the hadoop client user.",
+          SECURITY_AUTHENTICATION_TYPE, Constants.IMPERSONATION_HDFS_USER))
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
+          .setScope(Scope.CLIENT)
+          .build();
```

Comment:
as talked offline, do you want to make `IMPERSONATION_HDFS_USER` the default value ?

Reply:
Done.